lib/install/validate/validator.go

```diff
@@ -658,3 +663,48 @@ func (v *Validator) AddDeprecatedFields(ctx context.Context, conf *config.Virtua
 
 	return &dconfig
 }
+
+func (v *Validator) syslog(conf *config.VirtualContainerHostConfigSpec, input *data.Data) {
```

Comment:
feels like this is just url parsing for input parameter, could you move it to cmd package? 
I think validator is used for vsphere related object validation.